### PR TITLE
Henry/GitHub actions

### PR DESCRIPTION
**Pull request by henrybley**
#3 

added simple template of github actions for gradle. testing with this PR.

### CODE REVIEW

File: .github/workflows/gradle-build-test.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            contents: read
+
+        steps:
+            -   uses: actions/checkout@v4
+            -   name: Set up JDK 21
+                uses: actions/setup-java@v4
+                with:
+                    java-version: '21'
+                    distribution: 'corretto'
+            -   name: Setup Gradle
```
Comment (bavodaniels) [inline .github/workflows/gradle-build-test.yaml:23]: i think you dont need this step as you are using the wrapper, I could be wrong though

File: .github/workflows/gradle-build-test.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                uses: gradle/actions/setup-gradle@v3
+
+            -   name: Build with Gradle Wrapper
+                run: ./gradlew clean build
+    test:
```
Comment (bavodaniels) [inline .github/workflows/gradle-build-test.yaml:28]: gradle build will also run the tests so this job is not relevant

**Review by bavodaniels — CHANGES_REQUESTED** [review]

File: .github/workflows/gradle-build-test.yaml (hunk 1 above)
Comment (henrybley) [reply .github/workflows/gradle-build-test.yaml]: this is now fixed

File: .github/workflows/gradle-build-test.yaml (hunk 2 above)
Comment (henrybley) [reply .github/workflows/gradle-build-test.yaml]: also fixed

**Review by bavodaniels — APPROVED** [review]

**Pull request merged**